cmake/modules/LXQtTranslateTs.cmake

```diff
@@ -168,14 +168,16 @@ function(lxqt_translate_ts qmFiles)
         endif ()
         if (NOT EXISTS "${TR_TRANSLATION_DIR}/${TR_REPO_SUBDIR}")
             message(STATUS "Setting git repository in the translations dir '${TR_TRANSLATION_DIR}' ...")
+            if (EXISTS "${TR_TRANSLATION_DIR}/.git")
```

Comment:
@palinek It's not an optimization. It actually fails when pulling `lxqt-panel/panel` translations.
See it  [here](https://gist.github.com/luis-pereira/7417b15d893bd1203fe64e19b08c8905)